Sources/StreamChat/Models/Attachments/ChatMessageAttachment.swift

```diff
@@ -17,6 +17,11 @@ public struct ChatMessageAttachment<Payload> {
     /// The attachment payload.
     public var payload: Payload
 
+    /// The downloading state of the attachment.
+    ///
+    /// Reflects the downloading progress for attachments.
+    public let downloadingState: AttachmentDownloadingState?
```

Comment:
Yeah it seems we did what I described above for the model, so we probably can do the same for the LocalAttachmentState?

Reply:
Done